src/main/java/AddTasks.java

```diff
@@ -0,0 +1,75 @@
+
+public class AddTasks {
```

Comment:
Shall we use nouns for classes? Perhaps, we could consider using  Class name called: "TaskAddon".